diff --git a/node_modules/mineflayer/lib/plugins/digging.js b/node_modules/mineflayer/lib/plugins/digging.js
index 496cf63..ef519d9 100644
--- a/node_modules/mineflayer/lib/plugins/digging.js
+++ b/node_modules/mineflayer/lib/plugins/digging.js
@@ -242,6 +242,9 @@ function inject (bot) {
       enchantments = enchantments.concat(helmetEnchantments)
     }
 
+    if (block.material === 'incorrect_for_wooden_tool')
+      block.material = 'mineable/pickaxe'
+
     const creative = bot.game.gameMode === 'creative'
     return block.digTime(
       type,
